distribution/src/assembly/bin.xml

```diff
@@ -1357,6 +1357,16 @@
         </fileSet>
 
         <fileSet>
+            <directory>src/scripts/</directory>
+            <outputDirectory>${pom.artifactId}-${pom.version}/wso2/analytics/bin
+            </outputDirectory>
+            <includes>
+                <include>**/carbondump.bat</include>
```

Comment:
It's better to copy these files with permission included. Using  fileMode element etc. Else when executing they might not have necessary permissions